Documentation/Metrics/arangodb_scheduler_low_prio_queue_last_dequeue_time.yaml

```diff
@@ -9,5 +9,10 @@ exposedBy:
   - coordinator
   - dbserver
   - agent
+  - single
 description: |
-  Last recorded dequeue time for a low priority queue item.
+  Last recorded dequeue time for a low priority queue item, i.e., the amount of
+  time the job was sitting in the queue. If there is nothing to do for a long
+  time, this metric will be reset to zero.
+  A large value for this metric indicates that the server is under heavy load
+  and low priority jobs cannot be dequeued in a timely manner.
```

Comment:
Since this is only the queue time, I would think that we can sensibly write that approx. a second here should be considered excessive, since it indicates that a lot of queuing is done. And if this time shows a growing trend, then there is a definite danger that the queue becomes full and requests are declined. Maybe we can formulate it like this?